setup.py

```diff
@@ -171,6 +171,8 @@ def run(self):
     'rdflib==4.2.2',
     'rdflib-jsonld>=0.4.0',
     'requests>=2.21.0',
+    'rq==1.2.0',
```

Comment:
I think these and other service-specific requirements should _not_ be installed with the normal install of renku (i.e. the CLI). We should create a new "extras" here and handle those dependencies separately.

Reply:
Sure, but in that case we also need to extract other dependencies and put them in there as well (e.g. flask, marshmallow, etc). Since this is out of scope of this, I propose we open a ticket for this.

Reply:
yep, exactly, there are quite a few of them by now

Reply:
ah sorry didn't see your edit - yes, that's what I had in mind!

Reply:
I open a ticket for this so we can followup this separation there: #983